package.json

```diff
@@ -0,0 +1,133 @@
+{
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.2.0",
+    "csv-loader": "^3.0.3",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.24.2",
+    "html-webpack-plugin": "^5.3.2",
+    "json5": "^2.2.0",
+    "style-loader": "^3.2.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.20.1",
+    "toml": "^3.0.0",
+    "webpack": "^5.52.0",
+    "webpack-cli": "^4.8.0",
+    "xml-loader": "^1.2.1",
+    "yamljs": "^0.3.0"
```

Comment:
- [ ] All webpack related packages should be included in package.json like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as devDependencies. You are missing `webpack-dev-server`